### DISCUSSION TITLE
Can I disable scroll sync in Markdown editor?

### DISCUSSION THREAD

**Discussion by ideless** [requarks/wiki#6218] [Help / Questions] [answered]
Scroll syncing is generally a useful feature in Markdown editors, but there are situations where it doesn't work correctly or where I prefer to keep the preview section static. Unfortunately, I cannot find any option to adjust the editor settings in my admin settings page (I am using the v2.5.296 version of Wiki.js). It would be greatly appreciated if more editor configuration options were made available. Thank you in advance for any help!

**Comment by NGPixel** [comment, marked as answer]
This is not possible at the moment no.
